code/src/button.css

```diff
@@ -0,0 +1,205 @@
+
+.btn-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+ 
+button {
+  cursor: pointer;
+}
+
+
+/* ------------------ BUTTON START ANIMATION ------------------ */
+
+.button-start {
+	width: 200px;
+	height: 50px;
+	line-height: 50px;	
+	background-color: #f9e9d0;
+  border-radius: 8px;
+	color: #1d192b;
+  /* font-family: 'Indie Flower', cursive; */
+  font-size: 1.3rem;
+  font-weight: normal;
+	text-decoration: none;
+	text-align: center;
+	display: inline-block;
+  position: relative;
+  -webkit-transition-duration: 0.3s;
+  transition-duration: 0.6s;
+  -webkit-transition-property: -webkit-transform;
+  transition-property: transform;
+  -webkit-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+  box-shadow: 0 0 3px rgba(0, 0, 0, 0);
+  -webkit-transform: translateY(-6px);
+  -ms-transform: translateY(-6px);
+  transform: translateY(-15px);
+  -webkit-animation-name: hover;
+  animation-name: hover;
+  -webkit-animation-duration: 1.5s;
+  animation-duration: 1.5s;
+  -webkit-animation-delay: 0.3s;
+  animation-delay: 0.3s;
+  -webkit-animation-timing-function: linear;
+  animation-timing-function: linear;
+  -webkit-animation-iteration-count: infinite;
+  animation-iteration-count: infinite;
+  -webkit-animation-direction: alternate;
+  animation-direction: alternate;
+}
+
```

Comment:
This animation is awesome! You said you didn't have much time but you still managed to create this super fantastic button. Good job! 🥂